connectors/src/connectors/zendesk/temporal/activities.ts

```diff
@@ -432,6 +447,74 @@ export async function syncZendeskTicketBatchActivity({
   };
 }
 
+/**
+ * This activity is responsible for syncing the next batch of recently updated articles to process.
+ * It is based on the incremental endpoint, which returns a diff.
+ */
+export async function syncZendeskTicketUpdateBatchActivity({
+  connectorId,
+  brandId,
+  currentSyncDateMs,
+  cursor,
+}: {
+  connectorId: ModelId;
+  brandId: number;
+  currentSyncDateMs: number;
+  cursor: string | null;
+}): Promise<{ hasMore: boolean; afterCursor: string | null }> {
+  const connector = await _getZendeskConnectorOrRaise(connectorId);
```

Comment:
nit: don't name it `_functionName` if the function is not private

nit 2: (less important) I understand the envy to dry, I have the same. But:
- consistency with rest of codebase
- drying is low and slightly less clear than inlined

=> separate utils were not necessarily needed IMO in that specific case
(but feel free to not adress this one)

Reply:
I was hesitating on this one so I'll revert, thanks 👍